themes/default/content/docs/guides/continuous-delivery/gitlab-app.md

```diff
@@ -23,6 +23,11 @@ To enable the integration with your GitLab project, you will need to ensure you
   in Pulumi.
   * After you add the organization, ensure that it uses GitLab [as its identity provider]({{< relref "/docs/intro/pulumi-service/organizations#changing-identity-providers" >}}).
 
+{{% notes type="warning" %}}
+If you have [Merged Results Pipeline](https://docs.gitlab.com/ee/ci/pipelines/pipelines_for_merged_results.html) enabled for your project you must disable
```

Comment:
Why do they have to disable it? Is it for the reason you mention in the PR description? We should explain this in the doc as well.

Reply:
I was thinking that linking to the GitLab issue that explains the problem would be enough? This PR's description also doesn't explain the actual problem but the GitLab issue's description does. The explanation is a bit lengthy and I thought it would be too distracting to have in here. Let me know what you think.

Reply:
Here's what I mean. If I'm a GitLab user reading about this Pulumi feature and I might be interested in using it, and then I see this warning message telling me I have to go back to my GitLab project and disable a feature that I might actually be using, but not telling me why, I'm going to be confused. Why must I disable it? It is just that this particular Pulumi feature isn't compatible with it? If so, we should just say so: "This feature is currently not compatible with pipelines for merged results. See this issue for details as to why," or something similar.

Do you see the difference? Right now, we're just saying "you must disable this feature", and we're leaving it up to the user to go and figure that out why by reading the issue details and inferring that it must have something to do with the ephemeral Git SHA.

Reply:
> This feature is currently not compatible with pipelines for merged results. See this issue for details as to why

I like this sentence. I'll use this.

> which is probably not something they ever need to know about, anyway.

I mean...that's why I didn't want to explain it here. The linked issue does have a lot more context needed to understand why. Unless I reproduce the whole thing here I can't see how a small blurb about a pseudo-hash would be useful in this context.

Reply:
Yep, 💯 agree it's not useful info here, either.